mobile/www/js/index.js

```diff
@@ -33,19 +33,19 @@ function onDevice(){
            format: "json"
         }).done(function(data){
           config = data;
-          console.log(config);
-            apiURL = config['BLUEMIX_SERVER_URL'];
-        appId = config['NUTRITIONIX_APP_KEY'];
-        appKey = config['NUTRITIONIX_APP_ID'];
+          //console.log(config);
+        apiURL = config['BLUEMIX_SERVER_URL'];
+        appId = config['NUTRITIONIX_APP_ID'];
+        appKey = config['NUTRITIONIX_APP_KEY'];
         });
     
     
     
     var win = function (r) {
         
-    console.log("Code = " + r.responseCode);
-    console.log("Response = " + r.response);
-    console.log("Sent = " + r.bytesSent);
+    //console.log("Code = " + r.responseCode);
```

Comment:
i'm OK with leaving some logging in the code. I'd prefer that over leaving commented out code.